src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue

```diff
@@ -40,9 +40,12 @@ const timeSelectorModal = ref(null)
 
 const previewYear = computed(() => store.state.layers.previewYear)
 const hasMultipleTimestamps = computed(() => timeConfig.value.timeEntries.length > 1)
-
+const isTimeSliderActive = computed(() => store.state.ui.isTimeSliderActive)
+const isLayerVisible = computed(
+    () => !!store.getters.visibleLayers.find((layer) => layer.id === layerId.value)
+)
```

Comment:
This don't work as expected you can have duplicate layer don't search by id but by index.
The simplest is 
```javascript
const isLayerVisible = computed(() => store.state.layers.activeLayers[layerIndex.value].visible)
```

Reply:
This breaks if you have a duplicate layer, one which is visible and the other one not.

Reply:
Would be good to cover this in the e2e tests

Reply:
those are all good suggestions, and it should be done :)